hybris/egl/platforms/common/eglplatformcommon.cpp

```diff
@@ -111,13 +114,11 @@ extern "C" EGLBoolean eglplatformcommon_eglQueryWaylandBufferWL(EGLDisplay dpy,
 // and decRef'ed using eglHybrisReleaseNativeBuffer
 extern "C" EGLBoolean eglplatformcommon_eglHybrisAcquireNativeBufferWL(EGLDisplay dpy, struct wl_resource *wlBuffer, EGLClientBuffer *buffer)
  {
-     if (!buffer)
```

Comment:
Why is this check removed?

Reply:
it is covered by `server_wlegl_buffer_from` + `!buf` below

Reply:
All places which use buffer in this function there is dereference of it first with *buffer so if buffer pointer is null that is bad and the check should be added back.

Reply:
ah that's about `buffer` not `wlBuffer`, got it now.